tests/acceptance/10_files/zendesk_1779/zendesk_1779.cf

```diff
@@ -0,0 +1,40 @@
+bundle common test_meta
+{
+  vars:
+      "description" string => "Test file_selection attributes are only checked as necessary";
+      "story_id" string => "";
+      "covers" string => "";
+}
+
+# Ref: https://dev.cfengine.com/issues/6997
+
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent check
+{
+  meta:
+    "test_suppress_fail"
```

Comment:
Should be soft fail actually, since this is not a regression or a platform error. I fixed it for you.